cooperator/tests/test_mail_templates.py

```diff
@@ -263,6 +264,50 @@ def _test_mail_template_share_transfer(self, cooperator, subscription_request_va
             ),
         )
 
+    def _test_mail_template_share_transfer_all_shares(
```

Comment:
where is this function called? note that in this module there are `_test_*()` functions which contain common code for actual `test_*()` functions to avoid duplication.